packages/site/site-react/eslint.config.mjs

```diff
@@ -9,6 +9,8 @@ const config = [
     {
         rules: {
             "@comet/no-other-module-relative-import": "off",
+            "@calm/react-intl/missing-formatted-message": "off",
```

Comment:
Currently there is no `react-intl` package installed in `@comet/site-react. Unsure if we want internationalisation in this package.

Reply:
I believe this was intentional to not force a i18n library onto consumers of the packages.

Reply:
Currently we only have two string literals in `@comet/site-react` package, which are only used for development. so it should not be a problem.

e.g.:
<img width="839" alt="Screenshot 2025-06-12 at 08 46 33" src="https://github.com/user-attachments/assets/06cbb8ac-957a-4313-ae60-177234a74088" />
 for now.